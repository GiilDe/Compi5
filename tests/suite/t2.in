
int fib(int n) {
    if (n == 0 or n == 1) return 1;

    return fib((n-1)) + (fib(n-2));
}

void main() {
    printi(fib(4)); // Prints 3
    print("\n");
    printi(fib((7) - 1 / 1)); // Prints 8
    print("\nDone");
}