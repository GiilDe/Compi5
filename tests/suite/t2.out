3
8
Done